pkg/apis/servicecatalog/validation/bindingvalidation.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
If you're going to have the validations split like this, don't stutter; `validation/binding.go` would be fine.

I have no problem splitting into multiple files.